docs/REST API/rest-api.md

```diff
@@ -2346,7 +2346,7 @@ Returns the information of the specified host.
       "enrollment_status": null,
       "name": "",
       "server_url": null,
-      "macos_settings": {
+      "os_settings": {
```

Comment:
Hmm, this would be a breaking change. Which we could do but we have to be really intentional about it (check in w/ customers and users).

Also, I think we might have adding `os_settings` in addition to `macos_settings` in the BitLocker story: #12577 

If that's true, while it's not ideal to have 2 properties with duplicate information, I think we leave it as is.

Reply:
Agree.